section2/exercises/if-statements.js

```diff
@@ -43,7 +43,13 @@ Log to the console the appropriate age range
 
 var dogAge = 3;
 // Write your conditional here
-
+if (dogAge <= 2) {
```

Comment:
I like how you organized the conditionals to check here. It makes sense to eliminate the outliers first and then everything else falls into the "else" group.